srml/generic-asset/Cargo.toml

```
@@ -0,0 +1,28 @@
+[package]
+name = "srml-generic-asset"
+version = "0.1.0"
```

```suggestion
version = "2.0.0"
```
Make it align with Substrate version